ivy/functional/frontends/torch/tensor.py

```diff
@@ -1758,6 +1758,28 @@ def quantile(self, q, dim=None, keepdim=False, *, interpolation="linear", out=No
             self, q, axis=dim, keepdims=keepdim, interpolation=interpolation, out=out
         )
 
+    def random_(
+        self,
+        to=None,
+        *,
+        generator=None,
```

Comment:
why you are including `generator` and `kwargs`.

Reply:
I included `generator` as it is a key argument in the original function as seen in the docs. I believe it is required for testing. https://pytorch.org/docs/stable/generated/torch.Tensor.random_.html#torch.Tensor.random_

however `kwargs` was just my attempt at making the `from` in the docs somehow worked as I stated in my first comment.

Reply:
try to use `from_` instead of `from`.